tests/integration/integration_steps/fabric_steps.py

```diff
@@ -27,8 +27,12 @@ def delete_delta_table_data(credential: DefaultAzureCredential, path: str):
         print(f"Table not found {path}")
 
 
+# TODO: Rename function to be generic and not specific to time series
 def read_deltalake_timeseries(timeseries_path: str, credential: DefaultAzureCredential):
-    delta_table = get_ts_delta_table(credential, timeseries_path)
+    try:
+        delta_table = get_ts_delta_table(credential, timeseries_path)
+    except TableNotFoundError:
```

Comment:
should we log anything here in case someone used this function expecting data

Reply:
I'll add a logging message to make it easier to debug a test failure.  This function shouldn't be used outside of tests.